src/NetworkUtil.php

```diff
@@ -0,0 +1,345 @@
+<?php
+
+namespace Castor;
+
+class NetworkUtil
+{
+    /**
+     * Check if a port is accessible for a host.
+     *
+     * @param int         $port    The port to check
+     * @param string      $host    The host to check (default to 127.0.0.1)
+     * @param int         $timeout The timeout in seconds
+     * @param string|null $name    The name of the port (will be displayed in the output when $quiet is false)
+     * @param bool        $throw   If true, will throw an exception if the port is not accessible instead of returning
+     * @param bool        $quiet   If true, will not output anything
+     *
+     * @return bool True if the port is accessible, false otherwise
+     */
```

Comment:
Could you remove all un-necessary documentation? 
I mean, there is a lot of duplication between the PHPDoc and the PHP code with its type hint.